Makefile

```diff
@@ -86,6 +90,10 @@ dev-osx: docker-$(IMAGE)
 		-v $$GOPATH/src/github.com/fossas/fossa-cli:/home/fossa/go/src/github.com/fossas/fossa-cli \
 		fossa/fossa-cli:$(IMAGE) /bin/bash
 
+.PHONY: lint
+lint: $(GOLANGCI_LINT)
+	golangci-lint run
```

Comment:
I'm surprised the binary is being found. Is it actually being installed and used? (https://golangci.com/r/github.com/fossas/fossa-cli/pulls/508)